charts/growthbook/templates/deployment.yaml

```diff
@@ -57,6 +57,12 @@ spec:
             initialDelaySeconds: 30
             periodSeconds: 5
           env:
+            {{ if .Values.growthbook.sso.enabled }}
+            - name: LICENSE_KEY
```

Comment:
This looks like it will contain some secrets. If so it might need to go in as a secret value